addons/sale/models/sale_order_line.py

```diff
@@ -1316,7 +1316,7 @@ def _convert_to_sol_currency(self, amount, currency):
         return amount
 
     def has_valued_move_ids(self):
-        return self.move_ids
+        pass  # TODO: remove in master
```

Comment:
```suggestion
        return  # TODO: remove in master
```
if we expect the method to return something, it's good practice to use `return`
even if it does not change anything from using `pass` (like in this case)